pkg/services/login/login.go

```diff
@@ -109,10 +111,7 @@ func (ls *LoginService) UpsertUser(cmd *models.UpsertUserCommand) error {
 		}
 	}
 
-	err := ls.Bus.Dispatch(&models.SyncTeamsCommand{
-		User:         cmd.Result,
-		ExternalUser: extUser,
-	})
+	err := ls.TeamSync.SyncTeams(cmd.Result, extUser)
 	if err != nil && !errors.Is(err, bus.ErrHandlerNotFound) {
```

Comment:
```suggestion
	if err != nil {
```

Reply:
Will do it once remove the other bus handlers from the service in Enterprise.

Reply:
Huh, is there any way that `err` can be `bus.ErrHandlerNotFound` here?

Reply:
I thought yes, as long as the bus is still used inside of SyncTeams method in Enterprise. Do I get it wrong?